### PR TITLE
Docs for Inspector implementation status

### PR DESCRIPTION
**Pull request by logason**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…nspector-sdk-reference**](https://docs-9qjz6ihqu.avo-dev.app/implementation/avo-inspector-sdk-reference) | [**/workspace/inspector**](https://docs-9qjz6ihqu.avo-dev.app/workspace/inspector) |
|:-:|:-:|
|<a href="https://docs-9qjz6ihqu.avo-dev.app/implementation/avo-inspector-sdk-reference">Screenshot of /implementation/avo-inspector-sdk-reference</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/yLFfTlfUPYA_rKrcl1JX9LTyikMYWr6jIJrD3mKmv8-otOZFMoMZxFub7wbAeKZUMenDqFQUhyRDbS5L5FCeCA.png">(view full size)</a> |<a href="https://docs-9qjz6ihqu.avo-dev.app/workspace/inspector">Screenshot of /workspace/inspector</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/rPwUMugCZ2uDXf7w_gGCaezqk0dujhfYzqQtGLEKoS2w03rxp3fZAjoxpF2v1XnNxkH-Qj6u2Y7WP5Wotrk38g.png">(view full size)</a> |



| [**…/implementation-status**](https://docs-9qjz6ihqu.avo-dev.app/workspace/tracking-plan/implementation-status) |
|:-:|
|<a href="https://docs-9qjz6ihqu.avo-dev.app/workspace/tracking-plan/implementation-status">Screenshot of /workspace/tracking-plan/implementation-status</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/7NhYs99u0Q1MYEHpXLZ4UeEKYhjJVLeb6ytouTjgPT_brnZSJ7_6pcUAal4sq_BmOhtZTRE_Z-dCxqL7MMjz9w.png">(view full size)</a> |







Commit d94803e506fb48ae0a5b8e672641e96d6914baba (https://docs-9qjz6ihqu.avo-dev.app).

File: pages/implementation/avo-inspector-sdk-reference.mdx [hunk 1]
```diff
@@ -47,6 +47,25 @@ In order to ensure our SDK doesn't have a large impact on performance or battery
 In development batching is disabled by default.
 You can change the values using `AvoInspector.setBatchSize(newBatchSize: Int)` and `AvoInspector.setBatchFlustSeconds(newBatchFlushSeconds: Int)` class methods to fine tune the library performance.
 
+## <a name="versions"></a> App versions
+
+When initializing the Inspector SDK you'll be prompted to provide the current app version. Providing correct app version is not required, but doing so significantly improves the Inspector issue reporting. By providing app version the Inspector is able to tell in what version an issue was first seen, and in what version the issue was fixed. Versions also allow the Inspector to report more accurately on the implementation status of your events, by focusing on the implementation status in your latest app version for every source.
+
+We recommend using semantic versions **("1.0.0")** which get incremented on every release, but any type of incremental string or number will do (e.g. build number).
```
Comment (alex-tpom6oh) [inline pages/implementation/avo-inspector-sdk-reference.mdx:54]: A bit confusing here that numbers are in will do area and later are in great area, should we move numbers to recommended alongside semantic versioning?

**Comment by alex-tpom6oh** [comment]
Great docs!

File: pages/implementation/avo-inspector-sdk-reference.mdx (hunk 1 above)
Comment (logason) [reply pages/implementation/avo-inspector-sdk-reference.mdx]: Yes that's a good point, we don't care if it's semantic or not, as long as it increments. Updated in c55d6549cf515dab80b7449aec9f8eb5fa8b87dc!

File: pages/workspace/tracking-plan/implementation-status.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+
+### App versions
+
+By default the Inspector implementation status reports on how your tracking is performing in the latest version of your app. As we all know apps and websites can be running on multiple versions simultaneously, and since our tracking code changes with every release, the events can differ between versions. 
+
+If you have changed your event in the tracking plan, old versions of the event are expected to not match the current version of the tracking plan, and become noise. That’s why we built the Inspector Implementation Status to focus on the tracking from your latest versions
+
+We recommend using semantic versions **("1.0.0")** which get incremented on every release, but any type of incremental string or number will do (e.g. build number). Providing a version to the Inspector will help get the cleanest output from the status report and help you understand how your tracking is performing in your latest versions.
```
[the remark this answers is not in the document]
Comment (logason) [reply pages/workspace/tracking-plan/implementation-status.mdx]: Fixed in c55d6549cf515dab80b7449aec9f8eb5fa8b87dc

**Pull request merged**